var/spack/repos/builtin/packages/ghost/package.py

```diff
@@ -78,9 +75,12 @@ def cmake_args(self):
                 '-DGHOST_USE_ZOLTAN:BOOL=%s'
                 % ('ON' if '+zoltan' in spec else 'OFF'),
                 '-DBUILD_SHARED_LIBS:BOOL=%s'
-                % ('ON' if '+shared' in spec else 'OFF'), cblas_include_dir
+                % ('ON' if '+shared' in spec else 'OFF'),
+                '-DCBLAS_INCLUDE_DIR:STRING=%s'
+                % format(spec['blas:c'].headers.directories[0]),
```

Comment:
since you did not use any query parameters, then there is no need for `:c` here.